code for tinkering audio/tone_generation.py

```diff
@@ -0,0 +1,16 @@
+#function for generating a tone.
+def toneGeneration():
```

Comment:
The variables such as secs, freq, amplitude, etc. should be defined as [function arguments](http://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/functions.html#function-parameters).